README.md

```diff
@@ -28,6 +28,7 @@ Here’s the shopping list for this project. Depending if you’d like to crack
 * Bosch BME680 sensor with breakout board (see below for places to find one) or...
 * **Optional:** Sense HAT (optional replacement for the BME680, but does not include an air quality sensor)
 * **Optional:** 1-wire temperature sensor (temperature only)
+* **Optional:** [Pimoroni Enviro+ Air Quality](https://shop.pimoroni.com/products/enviro?variant=31155658457171)
```

Comment:
perhaps we should note that only  supports Temp, Humidity, Pressure, Light (Lux), NH3, Oxidising and Reducing Gases for now?

Reply:
@rahul-thakoor updated